tests/test_main.py

```diff
@@ -1,5 +1,4 @@
 import pytest
-
```

Comment:
This line is not extra.
We use it to separate related third party imports (pytest) from local application/library specific imports (app.main)

Generally, we have 3 categories of imports:
1. standard library imports
2. related third party imports
3. local application/library specific imports
And should separate all of them